deployment/template-cronjob.yaml

```diff
@@ -131,24 +131,36 @@ parameters:
   name: DAYS
   value: "7"
 
-- displayName: "Model bucket Name"
-  description: Name of the bucket which contains the model
-  required: true
-  name: AWS_BUCKET_NAME_MODEL
-  value: "stage-openshift-probable-vulnerability-models"
```

Comment:
What is the reason for removing model bucket config?

Reply:
We are not using that variable anywhere in template after your changes, so removed the same. Am I missing anything here?